### PR TITLE
Create basic tests and configure phpunit.xml.dist

### PR DESCRIPTION
**Pull request by malukenho**
- Create basic class for test Placephant\ResourceHelper class
- Create a simple file called phpunit.xml with configurations to unit tests with PHPunit

### CODE REVIEW

**Comment by erikaheidi** [comment]
.PHP Fatal error:  Class 'Helper\Resource' not found in /media/export/Projects/placephant/test/src/Helper/ResourceTest.php on line 40

can you fix this? :) i think its the only thing

**Comment by malukenho** [comment]
@erikaheidi path file changed!

File: phpunit.xml.dist [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit bootstrap="vendor/autoload.php" colors="true">
+    <testsuites>
+        <testsuite name="Helper">
+            <directory>test</directory>
+        </testsuite>
+    </testsuites>
+</phpunit>
```
Comment (cordoval) [inline phpunit.xml.dist:8]: run php-cs-fixer

File: test/src/Placephant/ResourceHelperTest.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            );
+        }
+    }
+
+    /**
+     * @test
+     * @dataProvider directory_provider
+     * @depends class_can_be_instantiated
+     */
+    public function class_can_storage_resource_path_on_resources_dir_property($path, $expected)
```
Comment (cordoval) [inline test/src/Placephant/ResourceHelperTest.php]: you can replace class for it_

Comment (cordoval) [inline test/src/Placephant/ResourceHelperTest.php]: or just go straight for the verb, can

File: phpunit.xml.dist [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit bootstrap="vendor/autoload.php" colors="true">
+    <testsuites>
+        <testsuite name="Helper">
```
Comment (cordoval) [inline phpunit.xml.dist]: it is not a Helper

File: phpunit.xml.dist (hunk 1 above)
Comment (cordoval) [inline phpunit.xml.dist:8]: you can use a tool like http://github.com/cordoval/gush

File: phpunit.xml.dist (hunk 3 above)
Comment (malukenho) [inline phpunit.xml.dist]: @cordoval  ResourceHelper's not a helper? .-.

Comment (cordoval) [inline phpunit.xml.dist]: here you should put i think the name of the project PlaceElephantImage Project or something

Comment (malukenho) [inline phpunit.xml.dist]: @cordoval  :+1:

**Comment by malukenho** [comment]
@erikaheidi I think it's old and can be closed, or you prefer I keep working on it?

**Pull request closed without merging**